### ISSUE TITLE
Firehol and ikev2 vpn

### ISSUE THREAD

**Issue by BbIKTOP** [firehol/firehol#323]
Pls add the possibility to configure ipsec vpn using firehol. Currently it's not possible to allow esp traffic in the INPUT/OUTPUT as described [here](https://serverfault.com/questions/900531/firehol-ipsec-configuration) 

Thank you

**Comment by philwhineray** [comment]
Can you determine exactly what is being blocked by looking at the iptables output as described [in the troubleshooting guide](https://firehol.org/guides/firehol-troubleshooting/).

If it is not obvious how to construct the rule, feel free to post some logs.

If there is no ipsec traffic blocked (but you others are correctly logged) then you most likely have a routing problem and not a firewall problem.

**Comment by philwhineray** [comment]
In fact, rereading, I see you proved it was not routing and have probably provided your own answer. If the connection appears to be direct to eth1, you need to put client entries on the eth1 interface for the services you want to access.

**Comment by BbIKTOP** [comment]
Sorry, I don't understand what do you want tot say. Could you pls explain?
Sure, I asked for such a possibility - to configure different access by different criteria, for example for different IPs. It's not possible now and it's a bug. It might be like a having several "interface" sections for the same interface but with the different parameters, like src and dst, or something else. As a sample why it might be necessary, I provided the ipsec ike tunnel, current industrial vpn standard
So, basically, there shall be the possibility to add something like that using firehol:

iptables -I INPUT  --match policy --pol ipsec --dir in  --proto esp  -s 10.1.0.0/16 -j ACCEPT
iptables -I OUTPUT --match policy --pol ipsec --dir out  --proto esp  -d 10.1.0.0/16 -j ACCEPT

**Comment by BbIKTOP** [comment]
Well, looks like I understood what do you mean. It’s sad that you answered not reading the description. It’s necessary to have different client entries for different source ips and protocols, that’s the question

**Comment by philwhineray** [comment]
What I find sad is that I spent time reading your description, trying to understand it and provide a useful response when it would have been better spent with my family.

I'm genuinely sorry you didn't find my answer helpful. Rest assured it won't happen again.

**Comment by BbIKTOP** [comment]
Sure. If you don't understand the question, or don't know the answer, or cannot explain it, you'd rather spent your time with your family then. I don't see it's not possible to allow all traffic FROM certain IPs for the connections that use certain protocols to be configured as a "client entries"
